### ISSUE TITLE
response to cocaine: exemplar of various issues with GO and behavior classes

### ISSUE THREAD

**Issue by cmungall** [geneontology/go-ontology#18807]
We have a hierarchy:

 - response to stimulus
    - [response to cocaine](http://amigo.geneontology.org/amigo/term/GO:0042220)
       - behavioral response to cocaine: Drd1, [Drd2](http://amigo.geneontology.org/amigo/gene_product/MGI:MGI:94924), etc (via IMP mostly)
       - cellular response to cocaine: Ehmt2 (via IDA mostly)

(some MGI annotations highlighted)

I have Questions

First, I think the two subclasses give false precision. In What sense are the dopamine receptor genes like Drd2 involved in _behavioral_ response, but not a _cellular_response? I think we are annotating the phenotype here. We already have an evidence code for IMP vs IDA, let's not make false precision in the ontology.

__A general gripe__: we have over-used the rule about annotating with as much specificity as you know and no more. Logically this is fine under an open-world assumption, but users don't make the OWA!!! If people come in with a query for "cellular response to cocaine" they will get a massively under-represented set. If we provide a term B that is a subclass of A, then it behooves us to be able to be able to do  reasonable job of making annotations more specific than A. 

Second, does this even belong in the ontology and is it correct?

Did Drd2 _evolve_ to respond to cocaine (either 'at the cellular level' (what other level??) or at the 'behavioral' level)? Is the organism genetically programmed to respond to cocaine? I don't think so. Yet, the definition of BP is _A biological process represents a specific objective that the organism is genetically programmed to achieve_.  And we have annotated a PAINT ancestral node to BRTC!

Regardless of whether it is programmed to or not, is Drd2 (or other dopamine receptors) actually involved? I think there is an interesting story here.  cocaine is able to exert part of its behavioral and cellular effect by elevating dopamine levels in the striatum https://www.ncbi.nlm.nih.gov/pubmed/19138563/ . It achieves this by binding to and inhibiting the presynaptic dopamine transporter https://www.ncbi.nlm.nih.gov/pubmed/18568020/ - so it doesn't seem appropriate for GO to annotate how chemicals disrupt parts of biological pathways... unless the organism has evolved a defense

However, another mechanism is via binding a receptor heteromer D1 and the σ1-receptor https://www.ncbi.nlm.nih.gov/pubmed/20956312 and this may be true for D2 as well... this does seem like a valid GO annotation, but to a complex.

But none of the biological story comes through in GO, just some over-annotated gene sets at inconsistent levels of granularity.

My recommendations:

 - be rigorous and consistent about our definition of BP _a specific objective that the organism is genetically programmed to achieve_
 - merge all 'level' specific subclasses of processes (e.g. cellular vs behavioral)
 - rethink how GO relates to behavior. See https://github.com/obo-behavior/behavior-ontology/issues/101. Perhaps GO shouldn't do behaviors at all. However, behavioral pathways may be in (e.g. mesolimbic dpoamine reward pathway)
 - don't use GO to house annotations that belong in an exposure database, a phenotype database, or a database of chemical-pathway interactions. Work with Monarch and the Alliance on a strategy for migrating these

**Comment by pgaudet** [comment]
Thanks @cmungall 

It would be nice to really be clear on how we handle phenotypes. This may be something useful but it certainly makes a lot of noise in GO annotations. 

I also don't understand 'response to drug' - other than detoxification, other responses are phenotypes/side effects, unlikely ti be 'evolved'. 

But where should we house those annotations? It seems there is a need. One option would be to allow to capture phenotypes directly, using different relationships (for example, in PMID:18032670 we could annotate that cocaine impairs GO:0007268 chemical synaptic transmission, or something along those lines. Annotating with different relations would allow us to only display 'normal functions' while still capturing phenotypes.  

(it's also very strange that response to cocaine is an 'adult response'). 

I'll remove the IBAs. (Generally we shouldn't have IBAs to 'responses'  terms).

**Comment by ValWood** [comment]
I agree with the  recommendations. Although the behaviour terms do amuse me.
